### PR TITLE
remove broken and fairly pointless deep sleep API

### PR DESCRIPTION
**Pull request by kilograham**
`multicore_reset_core1()` can be used instead

### CODE REVIEW

**Comment by kilograham** [comment]
fixes #77

**Review by liamfraser — DISMISSED** [review]

File: src/host/pico_multicore/include/pico/multicore.h [hunk 1]
```diff
@@ -13,29 +13,28 @@
...
-void multicore_sleep_core1();
 void multicore_launch_core1_raw(void (*entry)(void), uint32_t *sp, uint32_t vector_table);
 
-bool multicore_fifo_rvalid();
-bool multicore_fifo_wready();
+bool multicore_fifo_rvalid(void);
+bool multicore_fifo_wready(void);
 void multicore_fifo_push(uint32_t data);
 uint32_t multicore_fifo_pop_blocking();
 void multicore_fifo_drain();
```
Comment (lurch) [inline src/host/pico_multicore/include/pico/multicore.h:25]: Should this be updated to `void multicore_fifo_drain(void);` like you've done with the other functions? :shrug:

Comment (Wren6991) [reply src/host/pico_multicore/include/pico/multicore.h:25]: Yup I'll get that

**Review by Wren6991 — APPROVED** [review]
All looks good, this function was made redundant when we improved the bootrom sleep state before release, we just forgot to remove it.

**Pull request merged**